app/manager-frontend/src/helpers/dates.js

```diff
@@ -1,6 +1,61 @@
-export const getDateDaysAgo = (days) => {
+import { getDate } from "date-fns";
+
+export const getTimeHoursAgo = () => {
   // gets the date a specified amount of days ago
   var d = new Date();
-  d.setDate(d.getDate() - days);
+  d.setHours(d.getHours() - 2);
   return d;
 };
+
+const getDateString = (date, thisMonth = false, thisYear = false) => {
+  const options = {
+    year: "numeric",
+    month: "long",
+    day: "numeric",
+  };
+  const dateString = date.toLocaleString("default", options);
+  const hours = date.getHours();
+  let minutes = date.getMinutes();
```

Comment:
Nice one with the `toLocaleString` 👍 . Is it possible to the the times (hours and minutes) in a timezone aware fashion too? Presumably the `date` object you pass the function has that?

Reply:
I think it's already timezone aware, I'm passing it 'date' which is timezone aware and has the correct time offset. All this is doing is separating the hours and minutes so they can be displayed properly